src/utils/common.js

```diff
@@ -25,6 +25,9 @@ export const isNull = (value) => {
 };
 
 export const isDefinedAndNotNull = (value) => {
+  if (Array.isArray(value)) {
```

Comment:
nit: i believe this added check is semantically equivalent to the below. `createArrayFromValue` ensures `value` is an array or passed it back if it's already an array.